lib/membrane_element_ffmpeg_h264/decoder.ex

```diff
@@ -105,4 +110,12 @@ defmodule Membrane.Element.FFmpeg.H264.Decoder do
   end
 
   defp get_caps_if_needed(_, _, _), do: {:ok, []}
+
+  defp maybe_decode(nil, _decoder_ref), do: :ok
+
+  defp maybe_decode(data, decoder_ref) do
+    # ignore return of this function as ffpmeg treats it as an error. 
```

Comment:
Maybe waiting for the first buffer and prepending the parameter sets when it incomes would help avoiding that error?

Reply:
You're correct. I just tested that out, and it removes the error. Will update the PR

Reply:
@mat-hek i've updated the PR. Let me know what you think.